Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -493,6 +493,63 @@ theorem HasFPowerSeriesOnBall.comp_sub (hf : HasFPowerSeriesOnBall f p x r) (y :
       convert hf.hasSum hz using 2
       abel }
 
+open Pointwise in
+theorem HasFPowerSeriesWithinOnBall.comp_sub (hf : HasFPowerSeriesWithinOnBall f p s x r) (y : E) :
+    HasFPowerSeriesWithinOnBall (fun (z : E) ↦ f (z - y)) p (s + {y}) (x + y) r where
```

Comment:
```suggestion
    HasFPowerSeriesWithinOnBall (fun z ↦ f (z - y)) p (s + {y}) (x + y) r where
```
Not needed, the version without the ascription is slightly more readable IMHO. Same thing below.

Reply:
Could you the same for the other statements below?